followthemoney/schema/Document.yaml

```diff
@@ -112,10 +112,11 @@ Document:
       label: "Processing error"
       hidden: true
     processingAgent:
+      label: "Processing agent"
       label: "Name and version of the processing agent used to process the Document"
```

Comment:
Keeping the long version is fine I think, but you’d need to change the key to `description` ;)